lib/djvViewApp/PlaybackSpeedWidget.cpp

```diff
@@ -148,6 +166,49 @@ namespace djv
                         }
                     }
                 });
+
+            p.useCustomSpeedButton->setClickedCallback(
+                [weak]
+                {
+                    if (auto widget = weak.lock())
+                    {
+                        widget->_doSpeedCallback(widget->_p->customSpeed);
+                    }
+                });
+
+            p.customSpeedLineEdit->setTextEditCallback(
```

Comment:
As a note,  implementing `_doSpeedCallback()` in the `setTextEditCallback()` would cause a hard crash when changing the values in DJV.

Reply:
Hi Haryo, thank you for the contribution! I'm a bit swamped today trying to get the new release out, if it's OK I'll take a look at your changes over the weekend. Out of curiosity have you also tested out custom speeds with movie files in addition to image sequences?

Thanks, Darby

Reply:
I just checked in a fix for the crash, it would be nice to set the value in setTextEditCallback().